webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -625,9 +625,15 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
                                 {% endfor %}
                             </tbody>
                         </table>
-                        <p>
-                            <a href="#allthreads" data-show="Show other threads" data-hide="Hide other threads">Show other threads</a>
-                        </p>
+                        {% if parsed_dump.threads|length == 1 %}
+                            <p>
+                                No more threads to show.
+                            </p>
+                        {% else %}
+                            <p>
+                                <a href="#allthreads" data-show="Show other threads" data-hide="Hide other threads">Show other threads</a>
+                            </p>
+                        {% endif %}
```

Comment:
I don't know offhand if `.threads` could ever be `None` or some other non-int value or 0 or something like that. This doesn't take that into account. In those weird edge cases, it'll continue to show the "Show/Hide other threads" text.

Is there javascript that's looking for this `a` tag? Does it handle when it's not there at all?

Reply:
It works in my environment, but we have quite a lot of different modifications to original code and now I'm not sure it's a good idea to merge this without proper testing. I'll probably close it and reopen in case I have some crash to test this code. 
Sorry for wasting your time.